### DISCUSSION TITLE
auto health and mana

### DISCUSSION THREAD

**Discussion by morpheus092511** [diasurgical/DevilutionX#5468] [Ideas]
is there a possibility of adding a switch to auto use a health or mana potion when they get below a certain lvl

**Comment by qndel** [comment]
No, we already allow refilling belt with shift click and there's a setting that will autorefill belt potions from inventory. What's next, making the game play itself? :(

**Reply by morpheus092511** [reply]
Just started playing with devilutionX had not seen the refill from belt setting
But yes sometimes it fun just to run around and slash and kill everything Insight kinda like unlimited ammo on doom

**Reply by qndel** [reply]
we surely won't add such a thing, you can make a fork and add your own changes

![image](https://user-images.githubusercontent.com/14297035/200167862-9dc55f82-c060-4875-b661-8e11927f54ff.png)
